### ISSUE TITLE
[BUG] Profile picture cannot be changed after signup

### ISSUE THREAD

**Issue by yasharth291** [PalisadoesFoundation/talawa#185]
there is no way we can change the profile picture after signup. clicking the profile picture doesn't work at all

**Comment by pr4nshul** [comment]
Neither update profile button works.

**Comment by pr4nshul** [comment]
@yasharth291  are you working on this issue?

**Comment by CyberWake** [comment]
@yasharth291 are you working on it or will you be working upon it, if so could you assign this to me please. As there are lot of issues you are working upon.🙂

**Comment by yasharth291** [comment]
you have to ask from the mentors and let you assign this

**Comment by pr4nshul** [comment]
I think this issue should be closed now , as PR #446 will handle it once it is merged into master. @Sagar2366
